src/main/kotlin/net/ccbluex/liquidbounce/features/module/modules/player/nofall/modes/NoFallForceJump.kt

```diff
@@ -0,0 +1,79 @@
+package net.ccbluex.liquidbounce.features.module.modules.player.nofall.modes
+
+import net.ccbluex.liquidbounce.config.Choice
+import net.ccbluex.liquidbounce.config.ChoiceConfigurable
+import net.ccbluex.liquidbounce.event.events.PacketEvent
+import net.ccbluex.liquidbounce.event.handler
+import net.ccbluex.liquidbounce.features.module.modules.player.nofall.ModuleNoFall
+import net.minecraft.block.BlockState
+import net.minecraft.network.packet.c2s.play.PlayerMoveC2SPacket
+import net.minecraft.util.math.BlockPos
+import net.minecraft.util.math.Vec3d
+import kotlin.math.floor
+
+/**
+ * NoFallForceJump mode for the NoFall module.
+ * This mode forces the player to jump just when his about to land,
+ * preventing fall damage.
+ */
+internal object NoFallForceJump : Choice("Force Jump") {
+    override val parent: ChoiceConfigurable<*>
+        get() = ModuleNoFall.modes
+
+    private val blockDistance by int("Block Distance", 1, 0..5)
+    private val fallDistance by float("Fall Distance", 3.35f, 3.35f..10.0f)
+    private val jumpHeight by float("Jump Height", 0.42f, 0.1f..0.42f)
+    private var jumpTriggered: Boolean = false
+
+    /**
+     * Handles the packet event to check if a force jump should be triggered.
+     * This is done by checking if the player's fall distance is higher than the specific (fallDistance)
+     * and if the player is above a nonair block by the specific block distance.
+     */
+    val packetHandler = handler<PacketEvent> {
+        val packet = it.packet
+        if (packet is PlayerMoveC2SPacket && player.fallDistance > fallDistance) {
+            val blockBelowPos = calculateBlockBelowPosition()
+            val blockBelow: BlockState? = getBlockStateAtPosition(blockBelowPos)
+
+            if (!jumpTriggered && blockBelow != null && !blockBelow.isAir) {
```

Comment:
It would be better (and simpler) if you did this check with `FallingPlayer` (see https://github.com/CCBlueX/LiquidBounce/blob/nextgen/src/main/kotlin/net/ccbluex/liquidbounce/features/module/modules/player/nofall/modes/NoFallMLG.kt#L178)